pls check settings.py: DATABASE